src/components/organisms/code/deleteModal.tsx

```diff
@@ -3,45 +3,45 @@ import React from "react";
 import { useTranslation } from "react-i18next";
 
 import { ModalName } from "@enums/components";
-import { DeleteFile } from "@interfaces/components";
+import { DeleteModalProps } from "@interfaces/components";
 
 import { useModalStore } from "@store";
 
-import { Button } from "@components/atoms";
+import { Button, Loader } from "@components/atoms";
 import { Modal } from "@components/molecules";
 
-export const DeleteFileModal = ({ onDelete }: DeleteFile) => {
+export const DeleteFileModal = ({ isDeleting, onDelete }: DeleteModalProps) => {
 	const { t } = useTranslation("modals", { keyPrefix: "deleteFile" });
 	const fileName = useModalStore((state) => state.data as string);
 	const { closeModal } = useModalStore();
 
 	return (
-		<Modal name={ModalName.deleteFile}>
+		<Modal hideCloseButton name={ModalName.deleteFile}>
 			<div className="mx-6">
 				<h3 className="mb-5 text-xl font-bold">{t("title")}</h3>
 
-				<p>
-					{t("line1")} <strong>{fileName}</strong>
-				</p>
-				<p>{t("line2")}</p>
+				<p>{t("content", { name: fileName })}</p>
+
+				<p>{t("deleteWarning")}</p>
 			</div>
 
-			<div className="mt-14 flex justify-end gap-1">
+			<div className="mt-8 flex w-full justify-end gap-2">
 				<Button
 					ariaLabel={t("cancelButton")}
-					className="w-auto px-4 py-3 font-semibold hover:text-white"
+					className="px-4 py-3 font-semibold hover:bg-gray-1100 hover:text-white"
 					onClick={() => closeModal(ModalName.deleteFile)}
+					variant="outline"
 				>
 					{t("cancelButton")}
 				</Button>
 
 				<Button
 					ariaLabel={t("deleteButton")}
-					className="w-auto bg-gray-1100 px-4 py-3 font-semibold"
+					className="bg-gray-1100 px-4 py-3 font-semibold hover:text-error"
 					onClick={onDelete}
 					variant="filled"
 				>
-					{t("deleteButton")}
+					{isDeleting ? <Loader size="sm" /> : t("deleteButton")}
```

Comment:
It is better to leave the text  (`t("deleteButton")}`) and add the `Loader` if isDeleting = true

Reply:
done